requirements/base.in

```diff
@@ -5,7 +5,9 @@
 Django>=1.11,<2.3
 django-model-utils>=2.3.1
 
-# Use the same DRF version as edx-platform
-djangorestframework>=3.2.0,<3.10
 jsonfield2
 pytz
+
+
+# pinning it here to avoid any major changes.
+djangorestframework<3.12.0
```

Comment:
We'll probably be upgrading to DRF 3.12.2 very soon, and this constraint goes into `setup.py`.  Maybe change this to something like:

> \# Don't let edx-platform upgrade DRF past versions that have already been tested here
> djangorestframework<3.13.0